com.voxdsp.Borg.yaml

```diff
@@ -0,0 +1,27 @@
+app-id: com.voxdsp.Borg
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: borg
+rename-icon: borg
+rename-desktop-file: borg.desktop
+rename-appdata-file: borg.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: borg
+    buildsystem: simple
+    build-commands:
+      - cc main.c -Ofast -lSDL2 -lm -o borg
+      - install -Dm 0755 borg -t ${FLATPAK_DEST}/bin
+      - install -Dm 0644 flatpak/borg.desktop ${FLATPAK_DEST}/share/applications/borg.desktop
+      - install -Dm 0644 flatpak/borg.appdata.xml ${FLATPAK_DEST}/share/metainfo/borg.appdata.xml
+      - install -Dm 0644 flatpak/borg.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/borg.png
+    sources:
+      - type: git
+        url: https://github.com/BeverlyCode/Borg-ER-3
```

Comment:
Do you plan to maintain it as a full-fledged fork? I noticed it's been forked from https://github.com/mrbid/Borg-ER-3.

Reply:
Honestly, it's my repository and project, I am mrbid, so yeah I will maintain this as a full-fledged fork.

Reply:
verified

Reply:
> Honestly, it's my repository and project, I am mrbid, I will maintain this as a full-fledged fork.
> 
> Same for #3834

I can verify this statement.

Reply:
I will maintain this as a full-fledged fork.

Same for https://github.com/flathub/flathub/pull/3834

Reply:
> I will maintain this as a full-fledged fork.
> 
> Same for #3834

I can verify this statement.